### DISCUSSION TITLE
Bookmarks Lost after VSCode Update

### DISCUSSION THREAD

**Discussion by ambicoder** [alefragnani/vscode-bookmarks#587] [Q&A] [answered]
After every VScode update the bookmarks are lost...If I choose to save them in the progect instead of VSCode itself will this solve the problem?

**Comment by alefragnani** [comment]
Hi @ambicoder ,

I never received such _reincident_ report, neither noticed a single lost of bookmarks my own. I use 2 different machines with two different OS, with both Stable and Insiders release. It’s a bit surprise to me, and I wonder what the VS Code update process is doing wrong to wipe the bookmarks stored in workspace session. 

Just out of curiosity:

* Are you having similar issues with other extensions that does store _per project state_?
* Which OS and VS Code version are you using?
* Are you using Portable App approach?

But, about your question, if choosing to save the bookmarks in project could fix the issue, I guess/hope so, since the bookmarks will be stored within the project. Specially if you use com CVS like Git.

Hope this helps

**Comment by ambicoder** [comment]
I have not set for other extensions store per project state so I can tell you if I have similar issue or not.
I have WIn10 Home and my Vscode version is Version: 1.75.0 
About portable app approach....never heard of it.

**Comment by ambicoder** [comment, marked as answer]
yes...the last time vsode got updated the bookmarks were not lost...so setting the nookmarks per project solved the issue
